shared/utils/urls.ts

```diff
@@ -92,3 +93,13 @@ export function sanitizeUrl(url: string | null | undefined) {
   }
   return url;
 }
+
+export function urlRegex(url: string | null | undefined): RegExp | undefined {
+  if (!url || !isUrl(url)) {
```

Comment:
Lets use `sanitizeUrl` inside this method to make sure we're filtering out malicious urls from being embedded in an iframe

Reply:
Yes! Just realized this. Will update.

Reply:
Fixed!